test/studies/adventOfCode/2022/day11/bradc/day11-par.chpl

```diff
@@ -0,0 +1,181 @@
+use IO, List, Barriers;
+
+config const numRounds = 20;
+
+// things that define a monkey
+class Monkey {
+  var id: int,                 // an integer ID
+      Items: [0..1] list(int, parSafe=true),  // two lists of its items
+      current = 0, next=1,     // which list is the current vs. next one
+      op: owned MathOp,            // the operator the monkey does
+      divisor: int,            // the divisor for its % operation check
+      targetMonkey: 2*int,     // the target monkeys it throws items to
```

Comment:
If monkeys are a tuple, why not make `Items` a tuple too? You can still index into it using ints, I believe.

Reply:
I suspect I made it an array because I tend not to use tuples for modifiable data or complex data structures, perhaps due to years of being burnt by tuples not working very well in non-trivial cases?  Do you like it better as a tuple?

This comment is also making me realize I could do a better job of distinguishing var from const fields here...

Reply:
You already have fields.  How about just having two separate fields for the ifFalseTarget and ifTrueTarget?  You still need the array of lists because of ping-ponging between the old and new lists.

Reply:
I do prefer it as a tuple, I think, mainly because "fixed size array" is always preferable in my mind (even if `if{True,False}Targets` are made non-tuple).

Reply:
I spent the afternoon being bitten by a tuple bug in my parallel day12 code, so I'm back to being paranoid of tuples again...